examples/nvidia_dali/serialiize_dali_pipeline.py

```diff
@@ -0,0 +1,55 @@
+import json
+import os
+
+import nvidia.dali as dali
+import nvidia.dali.types as types
+
+
+def parse_args():
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--save", default="./model.dali")
+    parser.add_argument("--config", default="dali_config.json")
+    return parser.parse_args()
+
+
+@dali.pipeline_def
+def pipe():
+    jpegs = dali.fn.external_source(dtype=types.UINT8, name="my_source")
+    decoded = dali.fn.decoders.image(jpegs, device="mixed")
+    resized = dali.fn.resize(
+        decoded,
+        size=[256],
+        subpixel_scale=False,
+        interp_type=types.DALIInterpType.INTERP_LINEAR,
+        antialias=True,
+        mode="not_smaller",
+    )
+    normalized = dali.fn.crop_mirror_normalize(
+        resized,
+        crop_pos_x=0.5,
+        crop_pos_y=0.5,
+        crop=(224, 224),
+        mean=[0.485 * 255, 0.456 * 255, 0.406 * 255],
+        std=[0.229 * 255, 0.224 * 255, 0.225 * 255],
+    )
+    return normalized
+
+
+def main(filename):
+    config_file = open(args.config)
+    config = json.load(config_file)
+    batch_size = config["batch_size"]
+    num_threads = config["num_threads"]
```

Comment:
What's a good way to set this? Since each pipeline will run in a worker.

Also should pipelines be shared across workers?

Reply:
The above file serializes the pipeline and only the deserialized file is passed to MAR file along with dali_config.json.